test/integration/deploy_scale_test.go

```diff
@@ -15,7 +15,7 @@ import (
 
 	"github.com/openshift/api/apps"
 	appsapi "github.com/openshift/origin/pkg/apps/apis/apps"
-	appstest "github.com/openshift/origin/pkg/apps/apis/apps/test"
+	appstest "github.com/openshift/origin/pkg/apps/apis/apps/internaltest"
```

Comment:
I am gonna let naming of the import pass as that reduces the change set and is suppose to go away in a while. (Applies globally.)